### PR TITLE
feat: add route and njk template for a final comment submitted page

### PR DESCRIPTION
**Pull request by kandcdev**
## Ticket Number

https://pins-ds.atlassian.net/browse/AS-5752

## Checklist

- [ ] Requires infrastructure changes
- [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [ ] I have updated the documentation accordingly
- [ ] My commit history in this PR is linear
- [ ] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `main` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

File: packages/forms-web-app/src/routes/final-comment/final-comment.js [hunk 1]
```diff
@@ -14,5 +15,6 @@ router.post(
 	validationErrorHandler,
 	finalCommentController.postAddFinalComment
 );
+router.get('/final-comment-submitted', finalCommentSubmittedController.getFinalCommentSubmitted);
```
Comment (edpackard-work) [inline packages/forms-web-app/src/routes/final-comment/final-comment.js:18]: should this be in its own router file `/routes/final-comment/final-comment-submitted`?

File: packages/forms-web-app/src/views/full-appeal/submit-final-comment/final-comment-submitted.njk [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+{% block pageTitle %}
+  Final comment submitted - Appeal a planning decision - GOV.UK
+{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+			{{ govukPanel({
+        titleText: "Final comment submitted",
+				html: "Your reference number <br /><strong>0123456</strong>"
```
Comment (edpackard-work) [inline packages/forms-web-app/src/views/full-appeal/submit-final-comment/final-comment-submitted.njk:15]: does the reference number need to be a variable passed in from controller?

File: packages/forms-web-app/src/controllers/final-comment/final-comment-submitted.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+const { VIEW } = require('../../lib/views');
+
+exports.getFinalCommentSubmitted = async (_, res) => {
+	res.render(VIEW.FINAL_COMMENT.FINAL_COMMENT_SUBMITTED);
```
Comment (edpackard-work) [inline packages/forms-web-app/src/controllers/final-comment/final-comment-submitted.js:4]: do you need to send the case reference to the view? I believe the relevant part of the final comment object should be `finalComment.caseReference`

**Review by edpackard-work — CHANGES_REQUESTED** [review]
all fine, but you will probably want to move the router and add the functionality to render the case reference into the controller. If you look at https://pins-ds.atlassian.net/browse/AS-5814, you'll see what the final comment object looks like, and if you look at other recent PRs, you'll see how to handle the final comment object in your controller.

File: packages/forms-web-app/src/controllers/final-comment/final-comment-submitted.js (hunk 3 above)
Comment (edpackard-work) [reply packages/forms-web-app/src/controllers/final-comment/final-comment-submitted.js:4]: apologies Yan - following discussion with Alex, the reference should be `finalComment.horizonId`

Comment (edpackard-work) [reply packages/forms-web-app/src/controllers/final-comment/final-comment-submitted.js:4]: (and it's in the session, so `req.session.finalComment.horizonId`)

File: packages/forms-web-app/src/routes/final-comment/final-comment.js (hunk 1 above)
Comment (kandcdev) [reply packages/forms-web-app/src/routes/final-comment/final-comment.js]: done

File: packages/forms-web-app/src/views/full-appeal/submit-final-comment/final-comment-submitted.njk (hunk 2 above)
Comment (kandcdev) [reply packages/forms-web-app/src/views/full-appeal/submit-final-comment/final-comment-submitted.njk]: done

File: packages/forms-web-app/src/controllers/final-comment/final-comment-submitted.js (hunk 3 above)
Comment (kandcdev) [reply packages/forms-web-app/src/controllers/final-comment/final-comment-submitted.js]: done

**Review by edpackard-work — APPROVED** [review]

**Pull request merged**